templates/springboot/cloudevents/pom.xml

```diff
@@ -14,7 +14,7 @@
   <description>A Spring Cloud Function, using Cloud Events</description>
   <properties>
     <java.version>21</java.version>
-    <spring-cloud.version>2024.0.1</spring-cloud.version>
+    <spring-cloud.version>2023.0.5</spring-cloud.version>
```

Comment:
this.. does not make sense...?!?

Reply:
Yes, 2 PRs were opened, one proper one with a version bump, and this one, for some reason. A second PR with a version regression. I will look into in